.ci/Jenkinsfile

```diff
@@ -295,7 +295,11 @@ def withCloudTestEnv(Closure body) {
   }
   // Masking
   withEnvMask(vars: maskedVars) {
-    body()
+    withEnv(["TF_VAR_BUILD_ID=${BUILD_ID}",
+             "TF_VAR_BRANCH_NAME=${BRANCH_NAME}",
+             "TF_VAR_CREATED_DATE=${getCurrentBuildTime()}"]) {
+      body()
+    }
```

Comment:
I guess that this change isn't required as all TF_ vars are populated via `terraform_deployer.yml`